electron/channels/add-room-channel.js

```diff
@@ -1,20 +0,0 @@
-const { ipcMain } = require('electron')
```

Comment:
I was using "DELETE:" tag :)

Reply:
Yeah, this deleting should be done the same way as you delete the home stuff...

Reply:
Commit message change done.